### DISCUSSION TITLE
Error when try to click view message when clone a disabled channel

### DISCUSSION THREAD

**Discussion by IbrahimAlhaj9** [nextgenhealthcare/connect#5101] [General]
Hi Everyone, why I have this error:
![123](https://user-images.githubusercontent.com/54806435/158594571-1abf5595-b0a2-413d-91f0-7e246df9ae4b.PNG)

Steps:
1-Clone a disabled channel
2- Rename the New channel and click save
3- Go back to the channel lists and set the cursor on the cloned channel
4- click on the view messages button in the Channel tasks menu

--> Result: the error mentioned in the attached image is occur

Thank you

**Comment by pacmano1** [comment]
Likely because the metatdata columns are not in the db yet.  You can enable the channel, deploy it once (just don't have it poll on start), undeploy the channel and that should go away.   It should be more graceful of course, why don't you open an issue here on github for this problem.  You are in the discussions area at the moment, at the top you will see the issues link.   Do see if that has been posted as a problem already of course.

**Comment by IbrahimAlhaj9** [comment]
@pacmano1 thanks it's working now

**Reply by pacmano1** [reply]
Please mark things as answered when a posted answer works for you.  There should be a **button** on the github user interface for that.

**Reply by IbrahimAlhaj9** [reply]
@pacmano1 I don't have an answered button, I don't know why.
![Captur1e](https://user-images.githubusercontent.com/54806435/159653800-c8eb0568-1b52-4e85-895f-1e460fcb95f5.PNG)

**Reply by pacmano1** [reply]
Actually not your fault, I asked if Nextgen would change the "General" area to have the button.   The Q and A section does have that button.

**Reply by franklwambo** [reply]
Does the log file at C:\Program Files\Mirth Connect\logs point out to something meaningful. How about enabling and starting the new channel using the CLI guide at [https://forums.mirthproject.io/forum/mirth-connect/support/7582-mirth-cli-documentation](Mirth CLI). You might want to close the Mirth Connect administrator first, even if via the task manager.
